packages/deck/frontend/sirius-components-deck/src/card/DeckCard.tsx

```diff
@@ -49,8 +48,8 @@ export const DeckCard = ({
   editable,
   t: translate,
 }: DeckCardProps) => {
-  const updateCard = (card: Card) => {
-    onChange(card);
+  const updateCard = (updatedCard: UpdateCardProps) => {
```

Comment:
Is it `update` or `updated` card ?

Reply:
I will rename the argument cardToUpdate. Function will trigger the card update